cli/module_graph.rs

```diff
@@ -624,6 +633,10 @@ impl Default for BundleType {
 
 #[derive(Debug, Default)]
 pub struct EmitOptions {
+  /// If true, then code will be type checked, otherwise type checking will be
+  /// skipped.  If false, then swc will be used for the emit, otherwise tsc will
+  /// be used.
```

Comment:
Is it final that `tsc` will be used for emits if `check` is true or does https://github.com/denoland/deno/issues/8706 correspond to this bit? I think the former because we don't enforce `isolatedModules` in the runtime compiler API.

Reply:
It is the former at the moment (because #8706 is not resolved).  If we resolve #8706, I think we should consider ensuring that `isolatedModules: false` is not supported with this API.  We might actually want to consider it for this PR actually.  It will just cause people more problems than it fixes in supporting it.  We already have a slew of options we don't support for the runtime APIs.

Reply:
I've removed `isolatedModules` as a configurable option for the runtime API.  If someone needs `const enum` they will need to use `tsc` directly.

Reply:
Okay awesome! Thanks for clarifying.